src/packagers/SourcemapPackager.js

```diff
@@ -0,0 +1,38 @@
+const path = require('path');
+const sourceMap = require('source-map');
+const Packager = require('./Packager');
+const sourceMapUtils = require('../utils/SourceMaps');
+
+class SourcemapPackager extends Packager {
+  async start() {
+    this.generator = new sourceMap.SourceMapGenerator({
+      file: path.basename(this.bundle.name)
+    });
+  }
+
+  getOffsets(asset) {
+    let parent = this.bundle.parentBundle;
+    return parent.offsets.get(asset.relativename) || {line: 0, column: 0};
+  }
+
+  async addAsset(asset) {
+    if (asset.generated.map) {
```

Comment:
If an asset is added to a `map` bundle, it should definitely have a generated map. did you find otherwise?

Reply:
This is probably a remain from the initial approach, i'll remove this and test a bit to make sure it doesn't break